content/gherkin/reference.md

```diff
@@ -94,6 +95,9 @@ Free-form descriptions (as described above for `Feature`) can also be placed und
 
 You can write anything you like, as long as no line starts with a keyword.
 
+## Rule
+The Rule keyword is an optional keyword. It can be used to group scenarios that exemplify the same rule.
```

Comment:
Minor detail: please use back ticks around `Rule` here (for consistent formatting)

Reply:
Sure, done.